### ISSUE TITLE
Pricing on Cover Page

### ISSUE THREAD

**Issue by michaeldrescher** [18F/tts-buy-datagov-technical-support-services#43]
# Question

## Question/Comment
{ask away!}

The cover page of the RFQ is required to show the “total proposed price (including all options)” but the instructions also make clear that “The technical quotation must be included in a separate document from the price quotation…” If the price should be separate from technical, shall the offeror not include the price on the cover page for the technical proposal?
--

**Comment by oghaffari** [comment]
Yes please include the price on the cover page. The cover page can be submitted as a separate document.
